### DISCUSSION TITLE
Display node properties

### DISCUSSION THREAD

**Discussion by ChickenLegs01** [imohitmayank/jaal#54] [Ideas]
Hello Mohit,

Firstly I would like to say this is a great little project and works easy and fast.

My questions is:  
Would it be possible to add an extra display panel (or popup) that can display node properties of a selected node? 

Freddy

**Comment by imohitmayank** [comment]
Hey Freddy, we are exploring the possibility of popup. This should handle your requirement. The respective details can be found here: https://github.com/imohitmayank/jaal/issues/24

**Reply by ChickenLegs01** [reply]
Hello Mohit, Yes that would be good option. Thank you...
